### PR TITLE
[SPARK-25909] fix documentation on cluster managers

### PR DESCRIPTION
**Pull request by jameslamb**
## What changes were proposed in this pull request?

Propose changing the documentation to state that there are 4, not 3, cluster managers available.

## How was this patch tested?

This is a docs-only patch and doesn't need any new testing beyond the normal CI process for Spark.

### CODE REVIEW

File: docs/cluster-overview.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ There are several useful things to note about this architecture:
 
 # Cluster Manager Types
 
-The system currently supports three cluster managers:
+The system currently supports four cluster managers:
```
Comment (srowen) [inline docs/cluster-overview.md]: Heh OK. Let's just say 'several' to reduce future maintenance. Anything else that can be fixed in this doc?

Comment (jameslamb) [reply docs/cluster-overview.md]: ha good idea, I'll do that.

Nope everything else was great!

Comment (jameslamb) [reply docs/cluster-overview.md]: fixed

**Comment by srowen** [comment]
Merged to master/2.4

**Pull request closed without merging**